### PR TITLE
English version with permalinks (without redirections)

### PR DESCRIPTION
**Pull request by maukoquiroga**
Inspired on #734.

Allows to switch back and forth from english to french, and provides fallbacks, at least in the menu, when translations are not available.

### CODE REVIEW

**Comment by maukoquiroga** [comment]
(social graph tests fail also in my master branch 😕 )

**Comment by Morendil** [comment]
My bet would be a regression due to a recent Github Pages upgrade. Can you repro locally?

**Comment by maukoquiroga** [comment]
Yes ! I did a ˋ`ˋbundle update`ˋ` and fails locally.

**Comment by Morendil** [comment]
I've just merged a PR submitted recently, so my hypothesis above was incorrect. Not sure what's going on.

**Comment by Morendil** [comment]
The test failures I'm seeing seem to be caused by your changes local to that PR. For instance, the title of the home page should definitely not be "Français" as indicated in `_pages/fr/index.html` but `beta.gouv.fr` as specified in the test. Perhaps what is needed is a way to specify two distinct values, one for the page title and the other for the page's entry in the navigation menu.

File: _layouts/default.html [hunk 1]
```diff
@@ -6,17 +6,38 @@
         <header>
             <nav class="ui secondary menu">
                 <div class="right menu">
-                    {% if page.url != "/" and page.url != "/en/" %}
+                    {% if page.ref != 'landing' %}
                         <a class="item" id="logo" href="/">beta.gouv.fr</a>
                     {% endif %}
 
                     {% assign menu_items = site.pages | where_exp: 'page', 'page.menu_index > 0' | sort: 'menu_index' %}
```
Comment (Morendil) [inline _layouts/default.html]: This struck me as hard to understand, it took me several tries to get it. There seem to be contradictory assumptions: in displaying titles English is assumed to be the only language we have translations for (at least that's how I read it), whereas the "switches" portion allows for more than two languages.

In any case this is probably worth extracting to a separate file now.

**Comment by Morendil** [comment]
There are some valuable aspects about this PR: it creates a dedicated place to put non-blog content pages, and provides an organizing mechanism for multiple languages. It completely cleans the root level of the repository of content files - with the exception of the "rapport annuel" PDF and the RSS feed. (Can you find an equally satisfactory way of dealing with these two items?)

On the other hand I'm a bit concerned about the burden it places on people creating new pages, with three new mandatory front matter fields (`permalink`, `lang`, `ref`). I've invited @Flightan to look at the PR for an extra set of eyes.

File: _includes/head.html [hunk 2]
```diff
@@ -78,7 +78,7 @@
     <meta property="og:url" content="{{ canonical_url }}">
 
 
-    {% capture mastheadImagePath %}/img/{{ page.path | replace:'_posts','posts' | replace:'.html','' | replace:'.md','' }}.jpg{% endcapture %}
+    {% capture mastheadImagePath %}/img/{{ page.path | replace: '_pages', 'pages' | replace: '_posts', 'posts' | replace: '_startup', 'startup' | remove: '/fr' | remove: '/en' | remove: '.html' | remove: '.md' }}.jpg{% endcapture %}
```
Comment (maukoquiroga) [inline _includes/head.html]: Fixed a bug here.

File: _includes/head.html [hunk 3]
```diff
@@ -95,7 +95,7 @@
         @media (min-width: 2736px) {
             #masthead, footer::before {
                 background-image: url("{{ mastheadImagePath | replace:'.jpg','@4x.jpg'}}"),
-                                    url("{{ mastheadImagePath }}.jpg"); /* fallback */
+                                    url("{{ mastheadImagePath }}"); /* fallback */
```
Comment (maukoquiroga) [inline _includes/head.html]: Fixed another bug here.

File: _layouts/default.html (hunk 1 above)
Comment (maukoquiroga) [inline _layouts/default.html]: Yup, updated code.

File: _includes/head.html (hunk 2 above)
Comment (Morendil) [inline _includes/head.html]: Yay for unit tests :)

**Comment by Flightan** [comment]
This looks to me like a straightforward solution, I approve.
We need to find somewhere to document what each item in the header is used for.
Eg. in index.html:
```
---
additional_css:
- homepage
- dashboard
- startup-presentation
additional_js: homepage
permalink: /en/
lang: en
ref: landing
excerpt: We build digital public services
---
```

**Comment by Morendil** [comment]
Before merging:
- [ ] document page structure in Wiki page or CONTRIBUTING
- [ ] handle language switch link in posts as well as pages

**Pull request merged**